js/sbs_review.js

```diff
@@ -58,19 +58,18 @@ SideBySideReview.prototype = {
     },
 
     setReviewStatus: function (new_status, callback) {
-        var that = this;
-        $.post('/?a=set_review_status', {
-            review_id: this.review_id,
-            status: new_status
-        }, 'json')
-            .fail(function (data) {
-                that._saveFailureHandler(data);
-            })
-            .done(function (data) {
-                if (callback) {
-                    callback(data);
-                }
-            });
+        console.debug($.ajax(
+            '/?a=set_review_status',
+            {
+                type: 'POST',
+                data: {
+                    review_id: this.review_id,
+                    status: new_status
+                },
+                async: false
+            }).done(callback).fail(function () {
+            console.log('error');
+        }));
```

Comment:
I don't mind if you want to have some debug/error messages to be in the console for everyone but I think that in this case, you need to provide at least some more information. At least, I think it worth mentioning what's going on like
console.debug("ser_review_status", $.ajax ...)
the same stays for both console.debug and console.log calls